### PR TITLE
feat(suite-native): selected fee remembered

### LINKED ISSUE #15046: Send - Remember the transaction fee selected by the user
As a user during the "send flow, If I go back and forth in the flow without changing anything, I would like that my transaction speed that I have selected is remembered

Screenshot 2024-10-24 at 9 34 54 AM

### PR DESCRIPTION
**Pull request by PeKne**
## Description
- on every change of the selected fee level, it is saved to the send form draft right away
- if the user modify send outputs (adrress, crypto value, etc.) the selection is returned to the default `normal` level

## Related Issue

Resolve #15046 

## Screenshots:

https://github.com/user-attachments/assets/7747f4a3-5a5d-4cf5-956a-49b1c388f914

### CODE REVIEW

File: suite-native/module-send/src/sendFormThunks.ts [hunk 1]
```diff
@@ -275,7 +290,7 @@ export const calculateCustomFeeLevelThunk = createThunk(
         }
 
         const feeLevels = response.payload;
-        dispatch(sendFormActions.storeDraft({ accountKey, tokenContract, formState: draftCopy }));
+        // dispatch(sendFormActions.storeDraft({ accountKey, tokenContract, formState: draftCopy }));
```
Comment (vytick) [inline suite-native/module-send/src/sendFormThunks.ts]: Should this be commented? I it is not being used, please, delete

File: suite-native/module-send/src/sendFormThunks.ts [hunk 2]
```diff
@@ -221,6 +230,12 @@ export const updateSelectedFeeLevelThunk = createThunk(
         const draftCopy = { ...draft };
 
         draftCopy.selectedFee = feeLevelLabel;
+        if (feePerUnit) {
```
Comment (vytick) [inline suite-native/module-send/src/sendFormThunks.ts:233]: those if clauses are probably not needed here
```suggestion
            draftCopy.feePerUnit = feePerUnit;
            draftCopy.feeLimit = feeLimit;
```

**Review by vytick — COMMENTED** [review]
Looks nice, I have just a few nits

File: suite-native/module-send/src/sendFormThunks.ts (hunk 2 above)
Comment (PeKne) [reply suite-native/module-send/src/sendFormThunks.ts:233]: Unfortunatelly it is. The assigned values has to be string, but the arguments can be undefined.
Snímek obrazovky 2024-11-21 v 8 35 25

File: suite-native/module-send/src/sendFormThunks.ts (hunk 1 above)
Comment (PeKne) [reply suite-native/module-send/src/sendFormThunks.ts]: Forgotten leftover, removed in 991fea8f54

**Review by vytick — APPROVED** [review]
Nice job 🚢 🇮🇹

**Pull request merged**